app/db/crud/geography.py

```diff
@@ -14,6 +14,39 @@
 _LOGGER = logging.getLogger(__file__)
 
 
+def get_geo_subquery(
+    db: Session, allowed_geo_values=None, allowed_geo_slugs=None
+) -> Query:
+
+    geo_subquery = (
+        db.query(
+            func.min(Geography.value).label("value"),
+            func.min(Geography.slug).label("slug"),
+            FamilyGeography.family_import_id,
+        )
+        .join(FamilyGeography, FamilyGeography.geography_id == Geography.id)
+        .filter(FamilyGeography.family_import_id == Family.import_id)
+        .group_by(Geography.value, Geography.slug, FamilyGeography.family_import_id)
+    )
+    """ NOTE: This is an intermeadiate step to migrate to multi-geography support.
+    We grab the minimum geography value for each family to use as a fallback for a single geography.
```

Comment:
What do we mean by minimum in this context? The first geography value a family has?

Reply:
ah yes